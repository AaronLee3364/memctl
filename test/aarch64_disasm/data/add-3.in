fffffff0070c017c 8b2a46ca
