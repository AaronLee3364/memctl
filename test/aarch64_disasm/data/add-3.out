fffffff0070c017c  ADD     X10, X22, W10, UXTW #1
